staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -1585,3 +1589,9 @@ type CPUTopology struct {
 	// Must be a value greater or equal 1.
 	Threads uint32 `json:"threads,omitempty"`
 }
+
+type DiskIOThreads struct {
+	// Count specifies how many iothreads are allocated for the dedicated policy.
+	// +optional
+	Count *uint32 `json:"count,omitempty"`
```

Comment:
Do we maybe want to be more explicit about the relationship between count and the supplementalpool policy? `supplementalPoolThreadCount`?
I am struggling to find a native k8s/k8s example but something along these lines:
https://github.com/openshift/api/blob/d6d8306a24ec146d7e8cb7bdc1259a2144dc77a0/config/v1alpha1/types_image_policy.go#L161-L164

Reply:
Done, thanks!